### PR TITLE
Update 0000-01-03-BBSHSKumar.md

### PR DESCRIPTION
**Pull request by BBSHSKumar**
i am updating again

### CODE REVIEW

**Comment by BBSHSKumar** [comment]
updated the caption

**Comment by a-a-ron** [comment]
This is great!

File: _posts/0000-01-03-BBSHSKumar.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![one-does-not-simply](https://cloud.githubusercontent.com/assets/16547949/25400982/542b10d2-29c2-11e7-9cd6-f0d07042f1d2.jpg)
 {: .center}
 
-Caption here
+i am updating again
```
Comment (a-a-ron) [inline _posts/0000-01-03-BBSHSKumar.md:9]: This works well

**Review by a-a-ron — CHANGES_REQUESTED** [review]
This is great, but let's change the title on line 3 to say something else.

**Review by a-a-ron — APPROVED** [review]

**Comment by BBSHSKumar** [comment]
merged

**Pull request merged**